api/src/repositories/survey-repository.ts

```diff
@@ -380,33 +380,26 @@ export class SurveyRepository extends BaseRepository {
   }
 
   /**
-   * Get Occurrence submission id for a given survey ID
+   * Get Occurrence submission for a given survey id.
    *
    * @param {number} surveyId
-   * @returns {*} Promise<number>
+   * @return {*}
    * @memberof SurveyRepository
    */
-  async getOccurrenceSubmissionId(surveyId: number): Promise<number> {
+  async getOccurrenceSubmission(surveyId: number) {
```

Comment:
these should have a return, not obvious that it could be null

Reply:
updated